src/view/com/posts/PostFeed.tsx

```diff
@@ -335,6 +344,14 @@ let PostFeed = ({
 
   const {trendingDisabled, trendingVideoDisabled} = useTrendingSettings()
 
+  const ageAssuranceBannerState = useAgeAssuranceBannerState()
+  const selectedFeed = useSelectedFeed()
+  /**
+   * Cached value of whether the current feed was selected at startup. We don't
+   * want this to update when user swipes.
+   */
+  const isCurrentFeedAtStartupSelected = useRef(selectedFeed === feed)
```

Comment:
nit: probably better as state

```suggestion
  const [isCurrentFeedAtStartupSelected] = useState(selectedFeed === feed)
```